### DISCUSSION TITLE
Does this plugin use a MQTT broker and can it be used for other devices?

### DISCUSSION THREAD

**Discussion by x5nder** [nVuln/homebridge-lg-thinq#200] [Q&A]
Hey nVuln! Thanks for writing this plugin, it works perfectly for my LG air conditioning. I do have a question about MQTT: I see in the logs that the plugin features a built-in MQTT broker, is that correct? That would probably also explain why I can't install and run Mosquitto on my system on the default port (1883). Reason I am asking is that I have a new device in my network that features MQTT control and needs an MQTT broker for me to access it; does your broker allow non-LG devices or should I run Mosquitto on a different port in order to control non-LG devices?

**Comment by nVuln** [comment]
this plugin used MQTT as client only, not broker
